modules/test/test_archwiki.py

```diff
@@ -2,38 +2,77 @@
 test_archwiki.py - tests for the arch wiki module
 author: mutantmonkey <mutantmonkey@mutantmonkey.in>
 """
-
-import re
 import unittest
-from mock import MagicMock, Mock
+from mock import MagicMock
 from modules import archwiki
+import wiki
 
 
 class TestArchwiki(unittest.TestCase):
+
     def setUp(self):
         self.phenny = MagicMock()
+        self.input = MagicMock()
+
+        self.term = None
+        self.section = None
+
+    def prepare(self):
+        if self.section:
+            self.text = self.term + '#' + self.section
+            url_text = wiki.format_term(self.term) +\
+                '#' + wiki.format_section(self.section)
+        else:
+            self.text = self.term
+            url_text = wiki.format_term(self.term)
+
+        self.input.groups.return_value = [None, self.text]
+        self.url = 'https://wiki.archlinux.org/index.php/%s' % url_text
```

Comment:
Could you please use the new `format` method for this rather than the old style `%` replacement?